steps/firebase-app-distribution/1.0.0/step.yml

```diff
@@ -0,0 +1,111 @@
+title: Firebase App Distribution
+summary: Distribution pre-release version of your app to your trusted testers with Firebase App Distribution
+description: Firebase App Distribution gives a holistic view of your beta testing program across iOS and Android, providing you with valuable feedback before a new release is in production. You can send pre-release versions of your app using the console or your CI servers, and installing your app is easy for testers.
+website: https://github.com/d-date/bitrise-step-firebase-app-distribution
+source_code_url: https://github.com/d-date/bitrise-step-firebase-app-distribution
+support_url: https://github.com/d-date/bitrise-step-firebase-app-distribution/issues
+published_at: 2019-09-28T22:08:04.945934+02:00
+source:
+  git: https://github.com/d-date/bitrise-step-firebase-app-distribution.git
+  commit: 2e3120f6c72e583c63c67a18b51eaa67034d7e38
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
```

Comment:
Script uses npm to install dependency which is not recommended:
https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#do-not-use-submodules-or-require-any-other-resource-downloaded-on-demand

homebrew formula seems to be available: https://formulae.brew.sh/formula/firebase-cli
on Linux stacks it may become preinstalled: https://github.com/bitrise-io/android/pull/256/

Reply:
This is not official package from Firebase.
As following official way, I use cURL instead.
https://firebase.google.com/docs/cli#install-cli-mac-linux